content/blog/standup.md

```diff
@@ -0,0 +1,212 @@
++++
+title = "Standup Meeting Playbook"
+slug = "standup"
+date = 2023-12-28T17:48:58+05:30
+image = "/images/2023/12/meeting.jpg"
+draft = false
+authors = ["Akshay Vadher"]
+description = "Playbook to conduct effective standup meeting"
+tags = ["Playbook", "Software Craftsmanship", "Meeting", "Daily Scrum"]
+categories = ["Playbook", "Software Craftsmanship"]
+type = ""
++++
+
+The "Standup meeting" or "Daily Scrum" that everyone does religiously just for the sake of calling themselves "Agile."
+
+The name does not matter; you can call it anything. I like to call it “Huddle.”
+
+### Intent
+
+If we look at the [Agile Manifesto](https://agilemanifesto.org/), it does not describe how to manage your project,
+however, there are two methodologies that comes to our mind:
+
+1. [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)), which is the most popular; The "Daily Scrum" comes from that.
+2. [Extreme Programming](https://en.wikipedia.org/wiki/Extreme_programming) which is considered to be the best explanation of Agile.
+
+While “Standup meeting,” “Daily Sync meeting,” “Daily Scrum,” and any other name have different definitions, we should
+focus on the meeting from the Agile perspective instead of religiously following each one’s process and definition.
+
+Over the years, as “regular” project managers converted to the Agile way, Agile lost its core value, and so did standup
+meetings. Agile became just another name for the older project management process.
+
+### Values
+
+If we look at Agile values, it focuses on “Interactions,” “Collaboration,” “Delivery,” and “Change.” Remember these
+while creating your own rules for standup meetings.
+
+### FAQ
+
+##### Q: What is the ideal frequency?
+
+Usually, every day. However, if you are “Interacting” frequently and do not need to know the status, then you can meet less.
+
+##### Q: Where?
+
+Anywhere the whole team is comfortable. Physical or Remote. Does not matter.
+
+If you are conducting it remotely, always have videos on.
+You want to talk to humans, not animated figures!
+
+##### Q: When?
+
+Ideally at the start of the day. However, if everyone is not available or working remotely in different geographies, you
+can do it anytime.
+
+##### Q: What content?
```

Comment:
In this section can we write a little bit about how to keep it interesting. Typically we discuss these three things but to keep them interesting some teams at incubyte talks "I learned xyzzy" every Monday.
Or how was the weekend... etc

Reply:
It is mentioned at the end in the "Honorary addition" section.